classes/controllers/FrmFormsController.php

```
@@ -1713,10 +1713,11 @@ public static function update_button_classes( $classes ) {
 	/**
 	 * @param int|string $form_id
 	 * @param string     $class
+	 * @param mixed      $template_path
 	 *
 	 * @return void
 	 */
-	public static function mb_tags_box( $form_id, $class = '' ) {
+	public static function mb_tags_box( $form_id, $class = '', $template_path = 'default' ) {
```

PHPMD (2.15.0)

Avoid unused parameters such as '$class'. (undefined)

(UnusedFormalParameter)